sdks/java/core/src/main/java/org/apache/beam/sdk/runners/inprocess/InMemoryWatermarkManager.java

```diff
@@ -410,15 +414,17 @@ public Instant get() {
     /**
      * {@inheritDoc}.
      *
-     * <p>When refresh is called, the value of the {@link SynchronizedProcessingTimeInputWatermark}
+     * <p>
+     * When refresh is called, the value of the {@link SynchronizedProcessingTimeInputWatermark}
      * becomes equal to the minimum value of
      * <ul>
-     *   <li>the timestamps of all currently pending bundles</li>
-     *   <li>all input {@link PCollection} synchronized processing time watermarks</li>
+     * <li>the synchronized processing output timestamp of all currently pending elements</li>
+     * <li>all input {@link PCollection} synchronized processing time watermarks</li>
      * </ul>
      *
-     * <p>Note that this value is not monotonic, but the returned value for the synchronized
-     * processing time must be.
+     * <p>
+     * Note that this value is not monotonic, but the returned value for the synchronized processing
```

Comment:
This last note is confusing. "this value" = what? If it's the return, then it sounds like your saying "this is not monotonic but it must be".